src/tye/Program.RunCommand.cs

```diff
@@ -113,8 +119,8 @@ private static Command CreateRunCommand()
                 output.WriteInfoLine("Launching Tye Host...");
                 output.WriteInfoLine(string.Empty);
 
-                await using var host = new TyeHost(application.ToHostingApplication(), options);
-                await host.RunAsync();
+                await using var tyeHost = new TyeHost(application.ToHostingApplication(args.Host), options);
```

Comment:
I'm confused why we need to pass in the host here, shouldn't it be passed to the ApplicationFactory and set there?

Reply:
Fixed